source/functional-concepts.md

```diff
@@ -0,0 +1,150 @@
+# Functional Programming in Swift: A Primer
+
+Swift is a hybrid language that allows both objective-oriented and functional code to co-exist.
```

Comment:
All programming languages allow OO and functional code to co-exist. How about calling it "multi-paradigm, with strong influence from object-oriented and functional practices"?